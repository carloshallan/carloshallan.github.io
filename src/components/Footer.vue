<template>
  <footer>
    <p>Â© 2021 Carlos Hallan All rights reserved.</p>
  </footer>
</template>
<script lang="ts">
import Vue from 'vue'

export default Vue.extend({
  name: 'Footer'
})
</script>
<style lang="stylus" scoped>
footer
  background-color dark
  text-align center
  padding 20px

  p
    font-size 16px
</style>
