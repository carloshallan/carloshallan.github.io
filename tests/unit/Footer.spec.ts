import { mount } from '@vue/test-utils'
import Footer from '@/components/Footer.vue'

describe('Footer.vue', () => {
  it('Render Footer component', () => {
    const footer = mount(Footer)
    expect(footer.element).toMatchSnapshot()
  })
})
