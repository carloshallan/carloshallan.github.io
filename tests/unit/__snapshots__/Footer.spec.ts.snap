// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue Render Footer component 1`] = `
<footer>
  <p>
    Â© 2021 Carlos Hallan All rights reserved.
  </p>
</footer>
`;
